keras_core/ops/math.py

```diff
@@ -35,6 +35,31 @@ def call(self, data, segment_ids):
 
 @keras_core_export("keras_core.ops.segment_sum")
 def segment_sum(data, segment_ids, num_segments=None, sorted=False):
+    """Computes the sum of segments in a tensor.
+
+    Args:
+        data: Input tensor.
+        segment_ids: A 1-D tensor containing segment indices for each 
+                element in `data`.
+        num_segments: An integer representing the total number of 
+            segments. If not specified, it is inferred from the maximum
+            value in `segment_ids`.
+        sorted: A boolean indicating whether `segment_ids` is sorted. 
+                Default is `False`.
+
+    Returns:
+        A tensor containing the sum of segments, where each element 
+        represents the sum of the corresponding segment in `data`.
+
+    Example:
+    ```
+    >>> data = keras_core.ops.convert_to_tensor([1, 2, 3, 4, 5, 6])
+    >>> segment_ids = keras_core.ops.convert_to_tensor([0, 1, 0, 1, 0, 1])
+    >>> result = segment_sum(data, segment_ids)
+    >>> print(result)
+    tf.Tensor([9 12], shape=(2,), dtype=int32)
```

Comment:
Just print the values, no references to TF. E.g. you can just print `array([9 12], shape=(2,), dtype=int32)`